.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexRegex.cs

```diff
@@ -1,49 +1,54 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
+using System;
 using System.Collections.Generic;
+using System.Text;
 using System.Text.RegularExpressions;
 
-namespace Microsoft.Recognizers.Text.DataTypes.TimexExpression
+namespace ChitChattr.BotFramework.Recognizers
```

Comment:
Namespaces and class names should not be changed.

Reply:
same as above: copy-pasting late at night - this was just a working version from my own project, of course this should stay as it was